vignettes/get_data_taxonlijsten.Rmd

```diff
@@ -0,0 +1,427 @@
+The function `get_taxonlijsten_lists()` queries `D0156_00_Taxonlijsten` and gives an overview of all the taxonlists and taxonlist versions available in the database. Only the latest version is shown (unless specified otherwise).
+
+The following examples can be used as base to continue selecting the data you require.
+
+Get the most recent version of the 'Rode lijst van de Dagvlinders':
+
+```{r, echo=TRUE}
+rl_dagvlinders <- get_taxonlijsten_lists(con,
+                              list = "%rode%dagvlinders%",
+                              collect = TRUE
+                           )
+rl_dagvlinders
+```
+
+Get all recent versions of red lists. This time we will not use `collect = TRUE`, which will return a [lazy query](https://docs.lucee.org/guides/cookbooks/lazy_queries.html):
+
+```{r, echo=TRUE}
+rodelijst_recent <- get_taxonlijsten_lists(con,
+                              list = "%rode lijst%"
+                           )
+```
+
+Get all (most recent and older) taxonlist versions in the database:
+
+```{r, echo=TRUE}
+listversions_all <- get_taxonlijsten_lists(con,
+                              version = "all"
+                           )
+```
+
+Red lists need to be validated (= compliant with IUCN criteria) and authorized by the minister. Get the status of red lists:
+
+```{r, echo=TRUE}
+rodelijst_status <- get_taxonlijsten_lists(con,
+                              list = "%rode lijst%"
+                           )  %>%
+select("Taxonlijst", "PublicatieJaar", "Criteria", "Validering",
+       "Vaststelling")
+rodelijst_status
+```
+
+## Taxonlist features
+The function `get_taxonlijsten_features()` queries `D0156_00_Taxonlijsten` and gives an overview of all features associated with taxa on a taxonlist version. This is an auxiliary function to check the accepted values (KenmerkwaardeCodes) of the `feature` parameter in the core function `get_taxonlijsten_items`.
+
+Features are very context (list) dependent: it may be a red list status, an annex of a legislative list, a habitat, ... Not all lists have associated features, e.g. Soortbeschermingsplannen are (at this moment) featureless.
+
+The following examples can be used as base to continue selecting the data you require.
+
+Use function with default values (get all features of the most recent list versions):
+
+```{r, echo=TRUE}
+all_recent <- get_taxonlijsten_features(con)
+```
+
+Get features of all versions of the 'Rode lijst van de Dagvlinders':
+
+```{r, echo=TRUE}
+rl_butterfly <- get_taxonlijsten_features(con, version = "all",
+                                          list = "%rode%dagvlinders%",
+                                          collect = TRUE)
+```
+
+Get features of Habitattypical fauna:
+
+```{r, echo=TRUE}
+habitat_fauna <- get_taxonlijsten_features(con, list = "%Habitattyp%fauna%")
+```
+
+Note that the function also returns taxonlists without features:
+
+```{r, echo=TRUE}
+no_feature <- get_taxonlijsten_features(con, list = "%SBP%")
+```
+
```

Comment:
````suggestion

```{r, echo=TRUE, eval=TRUE}
no_feature
```
````
As you write 'note that the function returns ...', it seems relevant to show the output here.  ;-)